README.md

```
@@ -1,9 +1,183 @@
+docker-compose ps
+
+# 4. 로그 확인
+docker-compose logs -f
+
+# 5. 서비스 종료
+docker-compose down
+```
+
+### Docker Compose 구성
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)